Update mapping file upload limits (#26619)
DataDog/documentation: content/en/real_user_monitoring/guide/debug-symbols.md

If there are no debug symbols for your application, [upload them][2].

<div class="alert alert-warning">

> Ensure that the size of each debug symbol does not exceed the limit of **500 MB**, otherwise its upload will be rejected.

Reviewer: ```suggestion
Ensure that the size of each debug symbol does not exceed the limit of **500 MB**, otherwise the upload is rejected.
```

> For iOS dSYMs, up to **2 GB** are supported. 

Reviewer 2: ```suggestion
For iOS dSYMs, individual files up to **2 GB** are supported. 
```
